README.adoc

```diff
@@ -323,8 +323,8 @@ You can also access the `/health/ready` endpoint by going to the http://localhos
 Similarly, access the `/health/live` endpoint by going to the http://localhost:9080/health/live[http://localhost:9080/health/live^] URL to view the data from the liveness health check.
 
 Making code changes and recompiling is fast and straightforward.
-The development mode of Open Liberty automatically picks up changes in the `.class` files and artifacts, without needing to be restarted.
-Alternatively, you can run the `run` goal and manually repackage or recompile the application by using the `mvn package` command or the `mvn compile` command while the server is running. The development mode was added to further improve the developer experience by minimizing turnaround times.
+The dev mode of Open Liberty automatically picks up changes in the `.class` files and artifacts, without needing to be restarted.
```

Comment:
I'd recommend
The dev mode of Open Liberty
--->
Open Liberty dev mode

and

The dev mode was added...
-->
Dev mode was added..

Reply:
or, instead of "The dev mode of Open Liberty automatically picks up.."

"When Open Liberty runs in dev mode, it automatically picks up